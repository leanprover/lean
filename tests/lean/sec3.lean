section
  parameter A : Type
  definition tst (a : A) := a
  set_option pp.universes true
  check tst.{1}
end
