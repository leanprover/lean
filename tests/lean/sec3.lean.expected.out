sec3.lean:5:9: error: invalid use of explicit universe parameter, identifier is a variable, parameter or a constant bound to parameters in a section
tst : A â†’ A
